int32 size
hebi/HebiEntryMsg[] entries
